connectors/mongodb-connector/src/main/java/io/tapdata/mongodb/decoder/impl/AutoUpdateDateFilterTime.java

```diff
@@ -0,0 +1,118 @@
+package io.tapdata.mongodb.decoder.impl;
+
+import io.tapdata.entity.error.CoreException;
+import io.tapdata.mongodb.decoder.CustomSQLObject;
+import io.tapdata.util.DateUtil;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Map;
+
+public class AutoUpdateDateFilterTime implements CustomSQLObject<Object, Map<String, Object>> {
+    public static final String FUNCTION_NAME = "$dynamicDate";
```

Comment:
类名称与该常量一致，统一改为DynamicDate
```suggestion
public class DynamicDate implements CustomSQLObject<Object, Map<String, Object>> {
    public static final String FUNCTION_NAME = "$dynamicDate";
```